src/redux/store.ts

```diff
@@ -1,11 +1,13 @@
 import { Action, configureStore, ThunkAction } from '@reduxjs/toolkit'
 import createTaskReducer from './features/createTaskSlice'
 import taskBoardReducer from './features/taskBoardSlice'
+import filteredTaskBoardReducer from './features/taskBoardSlice'
 
 const store = configureStore({
   reducer: {
     createTask: createTaskReducer,
     taskBoard: taskBoardReducer,
+    filteredTaskBoard: filteredTaskBoardReducer,
```

Comment:
Is it necessary to add `filteredTaskBoard` reducer in the store from the same slice i.e. `taskBoardSlice`? 
This is not being used anywhere as we already have `taskBoardReducer` exported from the store.  Please explain your intention here if I didn't get it. @arpandhakal

Reply:
Yeah. I forgot to delete this. I used this in a previous implementation and it was left unattended. I will remove this thanks.